src/pages/repository.html

```diff
@@ -296,21 +296,51 @@ <h5>{{'repo.indexing.section' | translate}}</h5>
                         <div class="form-group row indented-div">
                             <label
-                                    class="col-xs-12 col-md-3 col-lg-2 col-form-label">{{'menu.ftsLanguages.label' | translate}}</label>
-                            <div class="col-xs-3 col-md-2" tooltip="{{'repoTooltips.ftsLanguages' | translate}}">
-                                <input class="form-control" id="ftsLanguages" type="text" name="ftsLanguages"
-                                       ng-model="repositoryInfo.params.ftsLanguages.value"
-                                       ng-disabled="repositoryInfo.params.fts.value === 'false'"/>
+                                    class="col-xs-12 col-md-3 col-lg-2 col-form-label">{{'menu.ftsIndexes.label' | translate}}</label>
+                            <div class="col-xs-3 col-md-2" tooltip="{{'repoTooltips.ftsIndexes' | translate}}">
+                                <input class="form-control" id="ftsIndexes" type="text" name="ftsIndexes"
+                                       ng-model="repositoryInfo.params.ftsIndexes.value"
+                                       ng-disabled="repositoryInfo.params.enableFtsIndex.value === 'false'"/>
+                            </div>
+                        </div>
+                        <div class="form-group row indented-div">
+                            <label
+                                class="col-xs-12 col-md-3 col-lg-2 col-form-label">{{'menu.ftsStringLiteralsIndex.label' | translate}}</label>
+                            <div class="col-xs-3 col-md-2" tooltip="{{'repoTooltips.ftsStringLiteralsIndex' | translate}}">
+                                <input class="form-control" id="ftsStringLiteralsIndex" type="text"
+                                       name="ftsStringLiteralsIndex"
+                                       ng-model="repositoryInfo.params.ftsStringLiteralsIndex.value"
+                                       ng-disabled="repositoryInfo.params.enableFtsIndex.value === 'false'"/>
+                            </div>
+                        </div>
+                        <div class="form-group row indented-div">
+                            <label
+                                class="col-xs-12 col-md-3 col-lg-2 col-form-label">{{'menu.ftsIrisIndex.label' | translate}}</label>
+                            <div class="col-xs-3 col-md-2" tooltip="{{'repoTooltips.ftsIrisIndex' | translate}}">
+                                <input class="form-control" id="ftsIrisIndex" type="text"
+                                       name="ftsIrisIndex"
+                                       ng-model="repositoryInfo.params.ftsIrisIndex.value"
+                                       ng-disabled="repositoryInfo.params.enableFtsIndex.value === 'false'"/>
+                            </div>
+                        </div>
+                        <div class="form-group row indented-div" style="display: none">
```

Comment:
This configuration will be in the next version, Should I remove it or I can leave it there until the next version

Reply:
I guess you can leave it, but I would suggest adding an `ng-if="false"` (so it does not show in the DOM at all) and a `TODO` inside, instead of the `display: none` style